packages/app/components/Nation/Territories/Territories.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { Icon, List, ListProps } from 'semantic-ui-react';
+import { compose } from 'recompose';
+
+import mockData from '../../mockData';
+import Routes from '../../../routes';
+import withCurrentDate, {
+  WithCurrentDateProps
+} from '../../decorators/withCurrentDate';
+import withEditMode, { WithEditModeProps } from '../../decorators/withEditMode';
+
+const Territories: React.SFC<
+  ListProps & WithCurrentDateProps & WithEditModeProps
+> = ({ currentDate, isEditMode }) => (
+  <List selection={true}>
+    {mockData.map(({ endDate: endDateFromData, id, startDate }) => {
+      // If no endDate is specified, we consider it today
+      const endDate = endDateFromData ? endDateFromData : new Date();
+
+      // Convert `startDate` to yyyy/mm/dd format
+      const url = startDate
```

Comment:
I've noticed that when clicking the territory evolution date cards (with the strange exception of the earliest card), the date is changed to one year before the start date. Should one year be added to this value?

Reply:
Fixed.